reference/lib/ReferenceOrderCombiner.sol

```diff
@@ -507,8 +512,11 @@ contract ReferenceOrderCombiner is
 
             // If offerer and recipient on the execution are the same...
             if (execution.item.recipient == execution.offerer) {
-                // increment total filtered executions.
-                totalFilteredExecutions += 1;
+                unchecked {
+                    // increment total filtered executions.
+                    // executions start at 0, infeasible to increment > 2^256.
```

Comment:
Nit: Capitalize Executions and place that word at the end of the comment line above.